### PR TITLE
Create / clear user cookie

### PR DESCRIPTION
**Pull request by joeylouise**
This PR:

Requires cookie-parser module
Adds dummy logout button with styling
Create cookie when user is authenticated
Clear cookie when user logs out
Relates #94

### CODE REVIEW

File: src/server/controllers/login.js [hunk 1]
```diff
@@ -19,8 +19,7 @@ module.exports = (req, res) => {
         res.send({ err: err.message });
       }
       if (isAuthenticated) {
-        // need to set cookie
-        res.redirect('/');
+        res.cookie('name', login, { maxAge: 3600000 }).redirect('/');
```
Comment (des-des) [inline src/server/controllers/login.js]: Is this one hour?

I would write it as 1000 * 60 * 60, otherwise it is hard to know what it means

**Review by des-des — CHANGES_REQUESTED** [review]
Okay, right now this is not secure, you have no way of verifying that a cookie returned sent by a client is authentic. I would use the cookie-session npm module. This will sign and verify your cookie.

**Comment by joeylouise** [comment]
@des-des I've changed this to use cookie-session and have set req.session.name when user logs in.

I've added in the session in `router.js` because it doesn't work when I add it in `app.js`

File: src/server/router.js [hunk 2]
```diff
@@ -8,7 +9,23 @@ router.use(bodyParser.urlencoded({
   extended: true,
 }));
 
+const sessionSecret = '3294r4whasuodifhd13urde80sufj138ryt9-gu538re9ghvisujela4hrsf4rue5r89';
```
Comment (des-des) [inline src/server/router.js]: No! Do not commit this. Put it in the env file!

File: package.json [hunk 3]
```diff
@@ -32,6 +32,8 @@
   },
   "dependencies": {
     "body-parser": "^1.17.2",
+    "cookie-parser": "^1.4.3",
```
Comment (des-des) [inline package.json]: Do you need this?

**Review by des-des — APPROVED** [review]
coolio looks good

**Review by des-des — CHANGES_REQUESTED** [review]
Ops see review above

**Review by des-des — APPROVED** [review]

**Pull request merged**